frontend/src/components/headerbar.tsx

```diff
@@ -0,0 +1,32 @@
+import styled from "styled-components";
+import { headerBarHeight, sidebarWidth, transitionDuration } from "../config";
+import { ISidebarAwareProps } from "./dashboard";
+
+/**
+ * Header bar for all views. Usually shows control to show/hide sidebar.
+ */
+export const HeaderBar = styled.div<ISidebarAwareProps>`
+  position: fixed;
+  display: flex;
+
+  top: 0px;
+  left: 0px;
+  width: 100%;
```

Comment:
Rather fix it to left and right, instead of having `100%` width. This can cause issues when the sidebar is open, as the width stays unchanged, but the container moves to the right.